src/hotspot/share/interpreter/oopMapCache.cpp

```diff
@@ -181,18 +181,6 @@ InterpreterOopMap::InterpreterOopMap() {
 #endif
 }
 
-InterpreterOopMap::~InterpreterOopMap() {
```

Comment:
Question: If we remove this opportunistic cleanup of `_bit_mask`, does it mean we might introduce memory inefficiencies in cases where `InterpreterOopMap` is not covered by close-by `ResourceMark`?

Reply:
In theory yes, although I doubt it is an actual issue because allocating in the resource area for this _bit_mask field is a rare case, and the memory allocated will be a few bytes. But I guess we can keep the eager cleanup just in case since it doesn't hurt.

Reply:
All right, your call. FWIW, this opportunistic cleanup is ugly, and I am happy to see it go.

Reply:
Agree.

Reply:
me too.